include/geometric_shapes/solid_primitive_dims.h

```diff
@@ -50,60 +50,61 @@ struct SolidPrimitiveDimCount
 };
 
 template <>
-struct SolidPrimitiveDimCount<shape_msgs::SolidPrimitive::SPHERE>
+struct SolidPrimitiveDimCount<shape_msgs::msg::SolidPrimitive::SPHERE>
 {
   enum
   {
-    value = static_cast<int>(shape_msgs::SolidPrimitive::SPHERE_RADIUS) + 1
+    value = static_cast<int>(shape_msgs::msg::SolidPrimitive::SPHERE_RADIUS) + 1
   };
 };
 
 template <>
-struct SolidPrimitiveDimCount<shape_msgs::SolidPrimitive::BOX>
+struct SolidPrimitiveDimCount<shape_msgs::msg::SolidPrimitive::BOX>
 {
   enum
   {
-    value =
-        (static_cast<int>(shape_msgs::SolidPrimitive::BOX_X) >= static_cast<int>(shape_msgs::SolidPrimitive::BOX_Y) &&
-         static_cast<int>(shape_msgs::SolidPrimitive::BOX_X) >= static_cast<int>(shape_msgs::SolidPrimitive::BOX_Z)) ?
-            static_cast<int>(shape_msgs::SolidPrimitive::BOX_X) :
-            (((static_cast<int>(shape_msgs::SolidPrimitive::BOX_Y) >=
-                   static_cast<int>(shape_msgs::SolidPrimitive::BOX_X) &&
-               static_cast<int>(shape_msgs::SolidPrimitive::BOX_Y) >=
-                   static_cast<int>(shape_msgs::SolidPrimitive::BOX_Z))) ?
-                 static_cast<int>(shape_msgs::SolidPrimitive::BOX_Y) :
-                 ((static_cast<int>(shape_msgs::SolidPrimitive::BOX_Z) >=
-                       static_cast<int>(shape_msgs::SolidPrimitive::BOX_X) &&
-                   static_cast<int>(shape_msgs::SolidPrimitive::BOX_Z) >=
-                       static_cast<int>(shape_msgs::SolidPrimitive::BOX_Y)) ?
-                      static_cast<int>(shape_msgs::SolidPrimitive::BOX_Z) :
-                      0)) +
-                1
+    value = (static_cast<int>(shape_msgs::msg::SolidPrimitive::BOX_X) >=
```

Comment:
What the hell is this doing? Just a comment - not related to this PR.

Reply:
It... attempts to find the highest index value between `BOX_X`, `BOX_Y`, `BOX_Z` and adds +1, ignoring that these are defined constants? I have no idea